api/envoy/service/orca/v1/orca.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package envoy.service.orca.v1;
+
+option java_outer_classname = "OpenRcaProto";
```

Comment:
Is this the right convention for classnames?

Reply:
Changed to `OrcaProto`